shared/src/hooks/useFetchSubgraphData.ts

```diff
@@ -63,6 +68,9 @@ const useFetchSubgraphData = (
                     : ""
                 }
                 ${vaultActivitiesGraphql(version)}
+                
+                # Token Account is indexed in v1 subgraph
```

Comment:
Are we sure we want to include comments in the query? Feels like it would add tiny amounts of latency